Pilot/dirac-pilot.py

```diff
@@ -1,21 +1,21 @@
 #!/usr/bin/env python
 
-""" The dirac-pilot.py script is a steering script to execute a series of
-    pilot commands. The commands may be provided in the pilot input sandbox, and are coded in
-    the pilotCommands.py module or in any <EXTENSION>Commands.py module.
-    The pilot script defines two switches in order to choose a set of commands for the pilot:
+"""The dirac-pilot.py script is a steering script to execute a series of
+pilot commands. The commands may be provided in the pilot input sandbox, and are coded in
+the pilotCommands.py module or in any <EXTENSION>Commands.py module.
```

Comment:
Not your fault, but it is actually 
```suggestion
the pilotCommands.py module or in any <EXTENSION>PilotCommands.py module (e.g. "LHCbPilotCommands.py"
```